tensorflow/core/util/ctc/ctc_beam_search.h

```diff
@@ -259,6 +259,16 @@ void CTCBeamSearchDecoder<CTCBeamState, CTCBeamComparer>::Step(
   } else {
     max_coeff = raw_input.maxCoeff();
   }
+
+  // Get normalization term of softmax: log(sum(exp(logit[j]-max_coeff))).
+  float logsumexp = 0.0;
+  for (int j = 0; j < raw_input.size(); ++j) {
+    logsumexp += expf(raw_input(j) - max_coeff);
```

Comment:
can you use std::exp and std::log?  or probably better, Eigen::numext::{log,exp}?